zcutil/build.sh

```diff
@@ -73,9 +76,17 @@ then
     shift
 fi
 
+# If --disable-rust is the next argument, disable Rust code:
```

Comment:
Hmm, in this case it might be necessary because of the `NO_RUST` argument to the depends system. So I'm non-blocking on this now, as it will be removed anyway once we make Rust a required dependency.